conda/signature.py

```diff
@@ -0,0 +1,66 @@
+import sys
+import base64
+import hashlib
+from os.path import abspath, expanduser, join
+
+from conda.compat import PY3
+
+from Crypto.PublicKey import RSA
+
+
+KEYS = {}
+KEYS_DIR = abspath(expanduser('~/.conda/keys'))
+
+
+def sig2ascii(i):
+    ret = []
+    while i:
+        i, r = divmod(i, 256)
+        ret.append(r)
+    if PY3:
+        s = bytes(n for n in ret[::-1])
+    else:
+        s = ''.join(chr(n) for n in ret[::-1])
+    return base64.b64encode(s).decode('utf-8')
+
+
+def ascii2sig(s):
```

Comment:
Please write a docstring that defines what is a valid input, and what an expected output should be.